dev/scss/_dashboard.scss

```diff
@@ -31,7 +27,7 @@
 .logo {
```

Comment:
Uhhhh no. This either its own module or it's `dashboard-logo`. Please clean up these files. I expect all of them to have the same prefix for each file ie. (`dashboard-blah`, `dashboard-content`), etc. This applies to all yoru files. continue reading your smacss book please. Thank you!